etcd/etcd-backup.adoc

```diff
@@ -1,7 +1,149 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="etcd-backup"]
 include::_attributes/common-attributes.adoc[]
-= Backing up etcd
+= Backing up and restoring etcd data
 :context: etcd-backup
 
-// This assembly will contain modules to provide information about backing up and restoring etcd.
\ No newline at end of file
+As the key-value store for {product-title}, etcd persists the state of all resource objects.
+
+Back up the etcd data for your cluster regularly and store it in a secure location, ideally outside the {product-title} environment. Do not take an etcd backup before the first certificate rotation completes, which occurs 24 hours after installation, otherwise the backup will contain expired certificates. It is also recommended to take etcd backups during non-peak usage hours because the etcd snapshot has a high I/O cost.
+
+Be sure to take an etcd backup before you update your cluster. Taking a backup before you update is important because when you restore your cluster, you must use an etcd backup that was taken from the same z-stream release. For example, an {product-title} 4.17.5 cluster must use an etcd backup that was taken from 4.17.5.
+
+[IMPORTANT]
+====
+Back up your cluster's etcd data by performing a single invocation of the backup script on a control plane host. Do not take a backup for each control plane host.
+====
+
+After you have an etcd backup, you can xref:../etcd/etcd-backup.adoc#dr-scenario-2-restoring-cluster-state-about_etcd-backup[restore to a previous cluster state].
+
+// Backing up etcd data
+include::modules/backup-etcd.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../hosted_control_planes/hcp_high_availability/hcp-recovering-etcd-cluster.adoc[Recovering an unhealthy etcd cluster for {hcp}]
+
+// Creating automated etcd backups
+include::modules/etcd-creating-automated-backups.adoc[leveloffset=+1]
+
+// Creating a single etcd backup
+include::modules/creating-single-etcd-backup.adoc[leveloffset=+2]
+
+// Creating recurring etcd backups
+include::modules/creating-recurring-etcd-backups.adoc[leveloffset=+2]
+
+[id="replace-unhealthy-etcd-member"]
+== Replacing an unhealthy etcd member
+
+This document describes the process to replace a single unhealthy etcd member.
```

Comment:
Somewhere deep in this list on the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=medium-global-audiences#style), it says to avoid self-referential text when possible. I normally don't care about instances like "the following procedures do X" but I guess something about "This document" stuck out to me.

I won't say it's required to rewrite, but could this maybe be rewritten to avoid such language? Or at the very least, maybe it could be more specific by saying "The following sections describe" rather than potentially referring to the entire assembly by accident

Reply:
Good point. I revised the wording to avoid the self-referential text.